### ISSUE TITLE
Can't focus on any data grid item with keyboard navigation

### ISSUE THREAD

**Issue by Jaiganeshkumaran** [files-community/Files#3557]
When using keyboard navigation, items list in list view layout mode is completely inaccessible.

**Comment by yaichenbaum** [comment]
@Jaiganeshkumaran I can't reproduce this issue in v1.2.2, what version of Files do you have?

**Comment by Jaiganeshkumaran** [comment]
Ok I can access it but unlike File Explorer it doesn't highlight focus so I didn't know

**Comment by yaichenbaum** [comment]
@Jaiganeshkumaran This should be fixed once WinUI creates the new DataGrid control.
